### ISSUE TITLE
Model expansion and optimization

### ISSUE THREAD

**Issue by MarynaSD** [Bootcamp-IA-P4/Proyecto-Clasificaci-n_grupo6#11]
Member 1: Gradient Boosting (XGBoost, LightGBM)
Member 2: Optimizing hyperparameters of base models
